examples/src/examples/user-interface/button-basic.tsx

```diff
@@ -47,7 +48,7 @@ class ButtonBasicExample extends Example {
         });
         app.root.addChild(screen);
 
-        // Create a simple button
+        /** Button */
```

Comment:
we don't use one liner /** */ comments in the examples .. perhaps keep it consistent.